stdm_config/views.py

```diff
@@ -189,25 +190,25 @@ def EntityLookupSummaries(profile, entity):
 
 def LooukupSummary(child_entity, ers):
 	parent_entity = ers[0].parent
-	#select count(*), cg.value from ko_farmer f left join ko_check_gender cg on cg.id = f.gender group by cg.value;
 	query = 'select count(*), ' + parent_entity.name + '.value from '+ child_entity.name + ' join ' + parent_entity.name + ' on ' + parent_entity.name+'.'+ers[0].parent_column + '='+ child_entity.name+'.'+ers[0].child_column+ ' group by '+ parent_entity.name + '.value;'
```

Comment:
1. Would suggest you use Python's f-string literal for string concatenation, using the '+' sign is not recommended. Will also make the code easier to read.
2. Check line length based on PEP8 guidelines, should be a max of 79 characters per line. You can also check for other parts of the code.

Reply:
This is noted. Thanks for the tip